serviceAccountKey.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
After MVP, we should really look into storing these securely. Fine for now, same with .env